README.md

```diff
@@ -1,15 +1,16 @@
 # RIF Relaying Services SDK sample dApp
 
-This a basic sample dApp to showcase how users can submit transactions to the RSK blockchain paying a RelayServer for the transaction fees with tokens. You will need to connect to the dApp with Metamask but only for signing transactions with the account that controls your Smart Wallets.
+This a basic sample dApp to showcase how users can submit transactions to the RSK blockchain paying a RelayServer for the transaction fees with tokens using RIF Relay SDK. You will need to connect to the dApp with Metamask but only for signing transactions with the account that controls your Smart Wallets.
 
-See https://github.com/rsksmart/enveloping for the RIF Enveloping project.
+See https://github.com/rsksmart/rif-relay for the RIF Relay project.
```

Comment:
it'd be good to format links in the standard markdown style (see [here](https://www.markdownguide.org/basic-syntax/#links))

Reply:
Fidex